Controls/Tests/Test_Cafe.c

```diff
@@ -16,15 +18,57 @@ int revenue = 0; // Business revenue, increased by the purchasing customers.
 
 int main(void){
 	OS_ERR err;
+	err = OS_ERR_NONE;
 
 	OSInit(&err);	// Initialize the OS
 	OS_CPU_SysTickInit();
 
 	// Create any semaphores/mutexes and initialize any global variables here
-	
+	OSMutexCreate(&RegisterOccupied_Mutex, "register mutex", &err);
+
+	// Initialize both customer 1 task
+	OSTaskCreate(
+		(OS_TCB*)&Customer_1_TCB, 
+		(CPU_CHAR*)"Customer 1", 
+		(OS_TASK_PTR)Task_Customer_1, 
+		(void*)NULL, 
+		(OS_PRIO)TASK_CUST_1_PRIO, 
+		(CPU_STK*)Task_Customer_1_Stk, 
+		(CPU_STK_SIZE)TASK_CUST_1_STACK_SIZE, 
+		(CPU_STK_SIZE)TASK_CUST_1_STACK_SIZE, 
```

Comment:
One small thing- for the first (CPU_STK_SIZE) argument, you probably want to use 
```
TASK_CUST_1_STACK_SIZE / 2
```
I had no idea what these parameters meant when I did the starter project and pretty much just copied them from code elsewhere, but as it turns out, the second ```(CPU_STK_SIZE)``` argument determines how large the stack is, and the first one sets a limit on how much of that stack to use. The extra stack space acts as a buffer because overflowing the stack means we could be accidentally overwriting someone else's data. Having a buffer just gives us an extra safety net. 

I think using the same value for both implies we're trying to use the whole stack as a buffer, though I'm not sure what the OS actually does to enforce/note this. As for Controls code, we usually use half the stack size for the buffer, so in the code you'll see ```WATERMARK_STACK_LIMIT```, which is defined as ```DEFAULT_STACK_SIZE/2```